### ISSUE TITLE
False positive in declaration-empty-line-before rule

### ISSUE THREAD

**Issue by chalkygames123** [kristerkari/stylelint-scss#492]
I'm not sure if this should open here or https://github.com/stylelint/stylelint, but I decided here because the problem only occurs with SCSS-specific syntax.

With the following configuration, a declaration followed by two consecutive double-slash comments is considered a violation by `declaration-empty-line-before` rule unexpectedly.

```
// .stylelintrc.js

module.exports = {
  rules: {
    'declaration-empty-line-before': [
      'always',
      {
        ignore: ['after-comment'],
      },
    ],
  },
}
```

Example SCSS code:

```
html {
  // foo
  // bar
  display: block;
}
```

When we remove either one of these comments, the error disappears.

I don't know if it's related, with the following code, the stylelint-disable comment is completely ignored, and thus an error reported.

```
html {
  // foo
  // stylelint-disable-next-line
  display: block;
}
```

My environment:

- stylelint v13.7.1
- stylelint-scss v3.18.0

Am I doing something wrong?

**Comment by chalkygames123** [comment]
It seems to have been fixed in https://github.com/stylelint/stylelint/pull/4913.
